src/screens/landing-page/landing-page.screen.tsx

```diff
@@ -1,3 +1,20 @@
+import { Box, Button, Image, Text } from 'components/primitive';
+
+import { WelcomeContainer, Title } from './landing-page.screen.styles';
+
 export const WelcomeScreen = () => {
-  return <>Welcome Screen</>;
+  return (
+    <WelcomeContainer hCenter vCenter>
+      <Box hCenter vCenter marginTop={6}>
+        <Image width={41.7} name='landingPage' />
+      </Box>
+      <Box hCenter vCenter>
+        <Box gap={0.8} hCenter vCenter>
+          <Title variant='title' text='Non custodial claimable wallets' />
+          <Text variant='small' text='A crypto wallet solution for easy self recovery and inheritance' />
+        </Box>
+        <Button variant='primary' label={{ text: 'Get Started' }} onClick={() => 'clicked'} color='primaryGradient' />
```

Comment:
Navigate to the login page on click of button. You can use `Link` or `Redirect` Reference: https://github.com/safient/wallet/blob/main/src/screens/home/home.screen.tsx

 <Redirect to={RoutePath.example} />

Reply:
Used `useHistory `hook here. using Link or redirect works, but for a simple button without data fetching logic, `history.push()` looks appropriate.